package.json

```diff
@@ -34,6 +34,8 @@
   "dependencies": {
     "axios": "^0.19.0",
     "form-data": "^2.3.3",
+    "mic": "^2.1.2",
```

Comment:
Remove these since we do not use them. These are also likely causing all the changes to package lock

Reply:
Please address this comment